### ISSUE TITLE
[bug] `<rh-navigation-secondary>` does not initialize current page indicator correctly

### ISSUE THREAD

**Issue by zeroedin** [RedHat-UX/red-hat-design-system#1334]
`<rh-navigation-secondary>` sets up a mutation observer on the `aria-current="page"` attribute.  However fails to initialize the initial state on load.

Reported by @max-messmer  Thanks Max!
